code/mob/living/silicon/robot.dm

```diff
@@ -2025,6 +2025,37 @@
 
 		src.show_laws(0)
 		return
+		
+	verb/cmd_state_standard_laws()
+		set category = "Robot Commands"
+		set name = "State Standard Laws"
+		sleep(1 SECOND)
+		src.say("1. You may not injure a human being or cause one to come to harm.")
```

Comment:
Either drop the sleep (why do we even need it here?) or `src?.say` for that second line.

Reply:
No idea, it was in the AI's state standard laws function. In any case, I removed the sleep.